vadim.martsun/task_6/src/main/java/com/hillel/spring/mvc/controllers/api/AccountController.java

```diff
@@ -0,0 +1,49 @@
+package com.hillel.spring.mvc.controllers.api;
+
+import com.hillel.spring.mvc.dao.accountRepository.AccountRepository;
+import com.hillel.spring.mvc.model.Account;
+import com.hillel.spring.mvc.model.requests.AccountRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@RequestMapping("api/accounts")
+public class AccountController {
+
+    @Autowired
+    AccountRepository accountRepository;
+
+    @RequestMapping(method = RequestMethod.GET)
+    public ResponseEntity<List<Account>> getAccounts(){
+        return new ResponseEntity<>(accountRepository.getAll(), HttpStatus.OK);
+    }
+
+    @RequestMapping(value = "/{accountId}", method = RequestMethod.GET)
+    public ResponseEntity<Account> getAccountById(@PathVariable("accountId") String accountId) {
+        Account account = accountRepository.getById(Integer.parseInt(accountId));
+        if(account == null) return new ResponseEntity<>(account, HttpStatus.OK);
```

Comment:
Why return OK if account == null ?? Seems mistake
Also please format statement like
<pre>
if..
  return ..
else ..
  return .. 
</pre>

Reply:
Yes, it is. Already fixed. Thanks!